### DISCUSSION TITLE
how do I configure the prisma extension in vscode?

### DISCUSSION THREAD

**Discussion by genzzo** [prisma/prisma#12411] [Q&A] [answered]
I would like to add some custom rules to my Prisma vscode extension (colors, spacing, etc.) 
The main one I'm interested in is having multiple spaces instead of one space when formatting
i.e (default):
```js
model User {
    id       String @id @default(uuid())
    username String @unique
    password String
}
```
would look more like:
```js
model User {
    id          String    @id @default(uuid())
    username    String    @unique
    password    String
}
```
Is this possible?

PS: I am aware of the existence of a prettier extension to configure Prisma but I prefer not to use it

**Comment by austincrim** [comment, marked as answer]
Hey @genzzo 👋 ,

We do not currently support that level of configuration with the Prisma extension. Feel free to open a [feature request](https://github.com/prisma/language-tools/issues/new?assignees=&labels=&template=feature_request.md&title=) on that repo.

It looks like the [Prettier plugin for Prisma](https://github.com/umidbekk/prettier-plugin-prisma) allows for configurable tab width, FYI.

**Reply by genzzo** [reply]
Will do. I don't really like to use prettier because of all the strict linting rules.
Thank you!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
